packages/landing-page/src/components/code-snippet.tsx

```diff
@@ -52,7 +52,7 @@ export function CodeSnippet() {
         <TabContent manager="bun" command="create solid" />
         <TabContent manager="npm" command="create solid" />
         <TabContent manager="yarn" command="create solid" />
-        <TabContent manager="deno" command="run -A npm:create-solid" />
+        <TabContent manager="deno" command="init --npm solid" />
```

Comment:
are you sure this is the right command? I think we want to use the create-solid cli?

Reply:
With the `--npm` flag, Deno automatically adds a `create-` to the command. I'm not a deno user, but I assume that's how it works based on their docs: https://docs.deno.com/runtime/reference/cli/init/#options-npm